### PR TITLE
refactor: log messages for unique fields

### PR DESCRIPTION
**Pull request by Shinigami92**
This is a addendum to #810

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "target": "ES5",
```
Comment (Shinigami92) [inline tsconfig.json:3]: I needed to set this explicit, so accessors are possible

File: docs/.vitepress/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
+    "target": "ES2020"
```
Comment (Shinigami92) [inline docs/.vitepress/tsconfig.json:3]: Somehow vitepress use target as build target and cannot handle const without that

**Review by ST-DDT — APPROVED** [review]
Does this need tests?

**Review by pkuczynski — APPROVED** [review]

File: src/unique.ts [hunk 3]
```diff
@@ -10,14 +11,74 @@ export class Unique {
...
+
+  /**
+   * Maximum time `unique.exec` will attempt to run before aborting.
+   *
+   * @deprecated Use options instead.
+   */
+  get maxTime(): number {
+    deprecated({
+      deprecated: 'faker.unique.maxTime',
+      proposed: 'Options',
```
Comment (pkuczynski) [inline src/unique.ts:24]: I am not sure if this is clear where those Options should be applied?

**Review by pkuczynski — APPROVED** [review]

File: src/unique.ts (hunk 3 above)
Comment (ST-DDT) [reply src/unique.ts:24]: Yeah, you are right. @Shinigami92 Please improve this.

Comment (Shinigami92) [reply src/unique.ts:24]: IMO it's okay, because there is only exactly one function that is related in the context with that.

**Pull request merged**